context.go

```diff
@@ -1,27 +1,28 @@
 package proxy
 
 import (
+	"context"
 	"net"
-
-	"github.com/getlantern/proxy/filters"
 )
 
 type contextKey string
 
 const (
-	ctxKeyUpstream     = contextKey("upstream")
-	ctxKeyUpstreamAddr = contextKey("upstreamAddr")
+	ctxKeyUpstream      = contextKey("upstream")
+	ctxKeyUpstreamAddr  = contextKey("upstreamAddr")
+	ctxKeyMITMTLSConfig = contextKey("mitmTLSConfig")
+	ctxKeyNoRespondOkay = contextKey("noRespondOK")
 )
 
-func upstreamConn(ctx filters.Context) net.Conn {
+func upstreamConn(ctx context.Context) net.Conn {
```

Comment:
I realized that this doesn't need the filters API, just plain context is enough.